idemp-base-test/src/org/bandahealth/idempiere/base/test/modelevent/OrderLineModelEventTest.java

```diff
@@ -0,0 +1,77 @@
+package org.bandahealth.idempiere.base.test.modelevent;
+
+import com.chuboe.test.populate.ChuBoeCreateEntity;
+import com.chuboe.test.populate.ChuBoePopulateFactoryVO;
+import com.chuboe.test.populate.ChuBoePopulateVO;
+import com.chuboe.test.populate.IPopulateAnnotation;
+import org.bandahealth.idempiere.base.model.MDocType_BH;
+import org.bandahealth.idempiere.base.model.MOrderLine_BH;
+import org.bandahealth.idempiere.base.model.MOrder_BH;
+import org.compiere.model.MInOut;
+import org.compiere.model.Query;
+import org.compiere.process.DocumentEngine;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class OrderLineModelEventTest extends ChuBoePopulateFactoryVO {
+	private ChuBoePopulateVO valueObject;
+
+	@IPopulateAnnotation.CanRunBeforeClass
+	public void prepareIt() throws Exception {
+		ChuBoePopulateVO valueObject = new ChuBoePopulateVO();
+		valueObject.prepareIt(getScenarioName(), true, get_TrxName());
+		assertThat("VO validation gives no errors", valueObject.getErrorMessage(), is(nullValue()));
+
+		valueObject.setStepName("Open needed periods");
+		ChuBoeCreateEntity.createAndOpenAllFiscalYears(valueObject);
+		commitEx();
+	}
+
+	@IPopulateAnnotation.CanRunBefore
+	public void createOrder() throws Exception {
+		valueObject = new ChuBoePopulateVO();
```

Comment:
I wanted to override it in case something in the `prepareIt` in case we start adding other test methods. The `CanRunBefore` runs before each test, so it should clear whatever was previously on the `valueObject`.